azure-intro/java/content/lab-02/README.md

```diff
@@ -0,0 +1,227 @@
+
+Now we have the resource group created, we can now add an Azure storage account to store our HTML content for our static site.
+
+We'll need to import the storage account resource first, so add the following to your imports, near the top of your `App.java`:
+
+```java
+import com.pulumi.azurenative.storage.StorageAccount;
+import com.pulumi.azurenative.storage.StorageAccountArgs;
+import com.pulumi.azurenative.storage.enums.Kind;
+import com.pulumi.azurenative.storage.enums.SkuName;
+import com.pulumi.azurenative.storage.inputs.SkuArgs;
+```
+
+Now we can use this import to create a storage account:
+
+```java
+var storageAccount = new StorageAccount("app", StorageAccountArgs.builder()
+        .enableHttpsTrafficOnly(true)
+        .kind(Kind.StorageV2)
+        .resourceGroupName(resourceGroup.name())
+        .sku(SkuArgs.builder()
+                .name(SkuName.Standard_LRS)
+                .build())
+        .build());
+```
+
+Notice a couple of things in the code we just wrote:
+
+1. We use the name of the resource group we created (`resourceGroup.name()`, a Pulumi output) as an input to the storage account we created. This allows the Pulumi engine to know that our resource group must be created before our storage account, and that any changes to the name of the resource group name must also be applied to the storage account. For more information, see [Inputs and Outputs](https://www.pulumi.com/docs/intro/concepts/inputs-outputs/).
+1. We can use Pulumi's enums to populate values without needing to look up the allowed values in the docs.
+
+## Step 3 &mdash; Create your Static Website
+
+Now we need to add a new resource, a `StorageAccountStaticWebsite`. This process is similar to before, so let's populate this resource.
+
+First, we need to add the following imports:
+
+```java
+import com.pulumi.azurenative.storage.StorageAccountStaticWebsite;
+import com.pulumi.azurenative.storage.StorageAccountStaticWebsiteArgs;
+```
+
+Add the following to your `App.java`:
+
+```java
+var staticWebsite = new StorageAccountStaticWebsite("app", StorageAccountStaticWebsiteArgs.builder()
+        .accountName(storageAccount.name())
+        .resourceGroupName(resourceGroup.name())
+        .indexDocument("index.html")
+        .error404Document("404.html")
+        .build());
+```
+
+## Step 4 &mdash; Upload your Static Files
+
+Our final step for our static website is to upload some files to it. In the previous step, we added a `StorageAccountStaticWebsite`. We can now upload some files to this using the container property.
+
+Again, we'll again need to add some imports first:
+
+```java
+import com.pulumi.azurenative.storage.Blob;
+import com.pulumi.azurenative.storage.BlobArgs;
+import com.pulumi.asset.FileAsset;
+```
+
+Because we're writing our infrastructure using Pulumi, which in turn uses real programming languages, we can use familiar mechanisms like loops, so let's loop through all the files in our sample app and upload them.
```

Comment:
Remember, these html files don't come as part of the template.. we need to tell the user they need to add them to their solution

Reply:
This threw me off, too. If you look at the `.gitignore`, you can see `my-first-azure-app` being ignored, and that the path of the files is `../wwwroot` in the sample code. The implication is that the reader is supposed to run the `mkdir` command from the root of the workshop tutorial.